lib/index.js

```diff
@@ -2247,10 +2277,9 @@ exposeMethod('zipAll0');
  * // => [ [ 1, 5, 9, 13 ], [ 2, 6, 10, 14 ] ]
  */
 
-Stream.prototype.zipAll = function (ys) {
-    return _([this]).concat(_(ys).map(_)).zipAll0();
-};
-exposeMethod('zipAll');
+addMethod('zipAll', function (ys) {
+    return this.create([this]).concat(_(ys).map(__(this.constructor))).zipAll0();
```

Comment:
Should be `this.create([this]).concat(this.create(ys).map(this.create)).zipAll0();`.